core-bundle/tests/Security/Voter/BackendAccessVoterTest.php

```diff
@@ -167,6 +167,12 @@ public function userDataProvider(): \Generator
             'text',
         ];
 
+        yield 'Check access on front end module' => [
+            ['frontendModules' => ['navigation']],
+            ContaoCorePermissions::USER_CAN_ACCESS_FRONTEND_MODULE_TYPE,
```

Comment:
We should replace `FRONTEND_MODULE` with `MODULE` and also `$frontendModule`, `frontendModules` etc. everywhere else, because we use `module` and `element` internally. The same constant for content elements is also called `USER_CAN_ACCESS_ELEMENT_TYPE` and not `USER_CAN_ACCESS_CONTENT_ELEMENT_TYPE`.

Reply:
The issue is, that `module` is internally used for back-end modules, too. The field `tl_user.modules` is already taken. This would need to be `frontendModules`. Otherwise I am fine with that.

Reply:
Same goes for the Constants in `ContaoCorePermissions`: There is already `USER_CAN_ACCESS_MODULE` for back-end modules

Reply:
Hm, that is unfortunate. @contao/developers What do you think? Should we use `USER_CAN_ACCESS_FRONTEND_MODULE_TYPE` then or maybe `USER_CAN_ACCESS_FE_MODULE_TYPE`?

Reply:
I would use `FRONTEND_MODULE`, not `FE`. We don't use abbreviations other than `DBAFS` imho. We already have `USER_CAN_ACCESS_FRONTEND_MODULES` for the theme permissions and we did not name it `FE` there either.

Reply:
That is true. 👍